algorithmic-strategies/divide-and-conquer.md

```diff
@@ -0,0 +1,292 @@
+# Divide & Conquer
+
+## Learning Goals
+
+By the end of this lesson we should be able to:
+
+- Explain what a divide-and-conquer solution is
+- Write some divide-and-conquer solutions
+
+## Overview
+
+**Divide and Conquer** is an approach to problem solving that breaks down a large problem into multiple, smaller subproblems. We use the results of those subproblems to solve the original problem.
+
+When we write a divide-and-conquer solution we normally:
+
+1. Break the problem into subproblems of the same type
+1. Recursively solve the subproblems
+1. Combine the solved subproblems to solve the larger problem
+
+This name is sometimes also applied to algorithms that only need to solve a single, reduced subproblem without the need to combine any results. In this case, the distinction between a general recursive algorithm, and one that might be called "divide and conquer" is largely based on how much we are able to divide the original problem at each recursive step.
```

Comment:
Aw, you noticed that this was a wishy-washy paragraph? Something rubs the wrong way about calling binary search and its ilk "divide and conquer" approaches. I can see why they are: it has to do with the constant rate of the decrease is proportional to the size of the data, rather than some fixed size, but I was having trouble phrasing that concisely, so punted. But you're version works well!